pkg/apis/openebs.io/v1alpha1/cstor_completed_backup.go

```diff
@@ -29,7 +29,28 @@ import (
 type CStorCompletedBackup struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
-	Spec              CStorBackupSpec `json:"spec"`
+	Spec              CStorCompletedBackupSpec `json:"spec"`
+}
+
+// CStorCompletedBackupSpec is the spec for a CStorCompletedBackup resource
+type CStorCompletedBackupSpec struct {
+	// BackupName is a name of the backup or scheduled backup
+	BackupName string `json:"backupName"`
+
+	// VolumeName is a name of the volume for which this backup is destined
+	VolumeName string `json:"volumeName"`
+
+	// SecondLastSnapName is a name of the second last completed-backup's snapshot
```

Comment:
cloud the code have used .snapName to store the last complete-backup snapname, and then use .prevSnapName to actually store the second last completed backup. 

In this case, though variable names are getting fixed, the actual values in the CR are still confusing. 

One way to fix this would be to deprecate the use of snapName and prevSnapName ( by making them omitifempty). 

Add new fields for lastCompletedSnapName, and secondLastCompleteSnapName (or call it penultimateCompletedSnapName)

Reply:
ok. I've updated https://github.com/openebs/maya/issues/1653 about updating etcd version and crd. I'll add upgrade job as part of this PR so that changes and upgrade goes into release.

Reply:
doc for upgrade approach https://docs.google.com/document/d/1kwCCYCFrcc51A7Xdt9fHmXOk32Z3J1FJQJyMPld9QbI/edit